bindings/matrix-sdk-crypto-ffi/src/lib.rs

```diff
@@ -28,7 +28,10 @@ pub use error::{
 use js_int::UInt;
 pub use logger::{set_logger, Logger};
 pub use machine::{KeyRequestPair, OlmMachine};
-use matrix_sdk_crypto::types::{EventEncryptionAlgorithm, SigningKey};
+use matrix_sdk_crypto::{
+    types::{EventEncryptionAlgorithm, SigningKey},
+    LocalTrust,
```

Comment:
Huh? Can we nowadays directly bind types from the `matrix-sdk-crypto` crate? No need to redefine them in the `matrix-sdk-crypto-ffi` crate and convert from one type to the other?

Reply:
This is how I always understood it to work, and the only reason to redefine was to have wrapped inputs/outputs (e.g. `Arc<BackupKeys>`. So yea theres probably a bunch of duplicate structs/enums we could remove

Reply:
Hmm, pretty sure we weren't able to implement some trait because the type nor the trait was part of our crate.

But if we can directly export the types from the `matrix-sdk-crypto` crate then that's great.